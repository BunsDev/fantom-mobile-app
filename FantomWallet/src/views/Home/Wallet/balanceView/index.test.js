import React from 'react';
import { shallow } from 'enzyme';

// eslint-disable-next-line import/first
import { BalanceViewContainer } from './index';

test('renders correctly', () => {
  const wrapper = shallow(<BalanceViewContainer balance="123.123" />);
  expect(wrapper).toMatchSnapshot();
});
