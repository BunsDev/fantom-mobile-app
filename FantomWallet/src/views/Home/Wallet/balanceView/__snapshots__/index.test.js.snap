// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignSelf": "center",
      "backgroundColor": "rgb(44,52,58)",
      "borderRadius": 10,
      "height": 150,
      "width": 718,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "rgb(50,59,66)",
        "borderTopLeftRadius": 10,
        "borderTopRightRadius": 10,
        "height": 40,
        "justifyContent": "center",
        "paddingLeft": 28,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "rgb(166,225,100)",
          "fontFamily": "SFProDisplay-Medium",
          "fontSize": 14,
        }
      }
    >
      Amount
    </Text>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <Text
      numberOfLines={1}
      style={
        Object {
          "alignSelf": "center",
          "color": "#fff",
          "flex": 1,
          "fontFamily": "SFProDisplay-Bold",
          "fontSize": 24,
          "textAlign": "center",
        }
      }
    >
      123.123
       FTM
    </Text>
  </View>
</View>
`;
